### PR TITLE
[rrd4j] Updated readme

### PR DESCRIPTION
**Pull request by JueBag**
Changed some wordings.

-Kept "DataSource" as the term for the way data is stored in a .rrd file. Such is more inline with the used "Default Datasources" (-numeric, ...).
-Removed the requirement for an everyMinute Strategy, inserted an explanation for relationship between Sample Intervall and Strategy instead.

Signed-off-by: Jürgen Baginski <opus42@gmx.de>

### CODE REVIEW

**Comment by JueBag** [comment]
Having spotted a "questionable" behaviour ([read last comment in here](https://github.com/openhab/openhab-addons/issues/8771)    )  I guess the remove of the everyMinute requirement has at least to be further discussed.

**Comment by andrewfg** [comment]
@JueBag please excuse me, but I am not really sure what your point / question is? Are you saying there is a bug in the code? Or in the documentation?

**Comment by JueBag** [comment]
I have to say sorry for interrupting you!
The code changes of rrd4j had a bug which got resolved (linked above). With this last change the readme got updated as well, however I had proposed some further changes readme as well.
I especially wanted to remove the comment about the everyMinute requirement. The tests I did today however revealed an odd behaviour which "could" be called a bug ( in order to keep the requirement removed) or just keep the requirement.

Since you did put some effort into the readme of rrd4j recently I figured you had an opinion on that which I wanted to incooperate.

File: bundles/org.openhab.persistence.rrd4j/README.md [hunk 1]
```diff
@@ -255,8 +258,7 @@ Items {
 ```
 
 **IMPORTANT:**
-The strategy `everyMinute` (60 seconds) **must** be used, otherwise no data will be persisted (stored).
-Other strategies can be used, too.
+When creating a custom datasource in the `rrd4j.cfg` file the used [sample Intervall](#sampleinterval-sample-interval) should be 20 seconds or less in order to keep the granularity. The selection of the used strategy has no effect on the granularity.
```
Comment (kaikreuzer) [inline bundles/org.openhab.persistence.rrd4j/README.md:261]:
```suggestion
When creating a custom datasource in the `rrd4j.cfg` file the used [sample interval](#sampleinterval-sample-interval) should be 20 seconds or less in order to keep the granularity. The selection of the used strategy has no effect on the granularity.  
```

**Review by kaikreuzer — APPROVED** [review]

**Comment by JueBag** [comment]
Thanks for merging!

Concerning the code change of the rrd4j persistence I would **suggest** a comment in the relaease notes (for the next milestone/ for the srtable release?) like:

```
Users of the **default** database setup of rrd4j Version 1 should note the changed setup for the now three different defaults. By the further use of rrd4j with no custom database setup the history would be lost!
In order to keep the history a custom setup, using the former default settings, is needed.

```

Problably the old default settings should be included as well.

**Pull request merged**